.github/workflows/tests.yml

```diff
@@ -6,7 +6,13 @@ on:
 
 jobs:
   tests:
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os:
+          - macos-12
```

Comment:
Should probably use latest for consistency:
```suggestion
          - macos-latest
```

Reply:
Good point, I copied it from one of the other workflows.

Reply:
When I think about it: maybe pin and be a bit more precise to the users on what OS versions this lib supports?

Reply:
Tests for Macos fail due to some lib missing. Did a bit of reading on how to solve it and maybe using `latest` is a solution here. Will give it a try.

Reply:
That seems to have done the trick 🎉